src/storage/store/table_copy_utils.cpp

```diff
@@ -507,6 +508,13 @@ std::unique_ptr<arrow::PrimitiveArray> TableCopyUtils::createArrowPrimitiveArray
     return std::make_unique<arrow::PrimitiveArray>(type, length, buffer);
 }
 
+void TableCopyUtils::validateStrLen(uint64_t strLen) {
+    if (strLen > BufferPoolConstants::PAGE_4KB_SIZE) {
+        throw CopyException(
+            ExceptionMessage::overLargeStringValueException(std::to_string(strLen)));
```

Comment:
At this point just inline the function then.

Reply:
We don't inline any function that has more than one line

Reply:
That's not what I mean. I mean, delete `overLargeStringValueException`, and here, do:

```cpp
throw CopyException(stringFormat("Maximum length of strings is 4096. Input string's length is {}.", strLen))
```

Reply:
We not include the exception message in the check function?

Reply:
Sorry I don't know what you mean by that... I've posted a suggested change that might clear up confusion.